test/Common/Microsoft.Test.OData.DependencyInjection/ContainerBuilderHelper.cs

```diff
@@ -5,21 +5,23 @@
 //---------------------------------------------------------------------
 
 using System;
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.OData;
+using Microsoft.OData.Core;
 
 namespace Microsoft.Test.OData.DependencyInjection
 {
     public static class ContainerBuilderHelper
     {
-        public static IServiceProvider BuildContainer(Action<IContainerBuilder> action)
+        public static IServiceProvider BuildContainer(Action<IServiceCollection> action)
         {
-            IContainerBuilder builder = new TestContainerBuilder();
-
-            builder.AddDefaultODataServices();
+            var builder = new TestContainerBuilder();
+            IServiceCollection services = builder.Services;
+            services.AddDefaultODataServices();

```

Comment:
I think we can just call `new ServiceCollection()`  and `services.BuildServiceProvider()` directly and get rid of `TestContainerBuilder`.